packages/flutter/lib/src/material/bottom_sheet.dart

```diff
@@ -216,14 +216,14 @@ class _BottomSheetState extends State<BottomSheet> {
 
   void _handleDragUpdate(DragUpdateDetails details) {
     assert(widget.enableDrag);
-    if (_dismissUnderway)
+    if (widget.animationController == null || _dismissUnderway)
```

Comment:
I think we should also check against `BottomSheet.enableDrag` here. If that is true then it is broken by this change since the drag to dismiss will not work.

Reply:
enableDrag can't work well with null-controller. 

In fact, it is rarely used this widget directly, this patch just to avoid crashing.

Reply:
True, it just swallows an error though that the developer should know about. If `enableDrag` is true, but the animation controller is not set, it should probably assert. Otherwise the drag functionality is broken and the developer won't even know it.

Reply:
That's right. I think some documents may also be needed here to explain the working mechanism of enableDrag.

If we use this widget directly and even we give a controller to it, the drag-to-dismiss isn't work!